### PR TITLE
blog: scheduler plugin reusing

### PR DESCRIPTION
**Pull request by uniemimu**
A blog post about scheduler plugin reusing as a followup of scheduling-sig discussions. In its own folder, because of the svg-images that are used.

### CODE REVIEW

File: content/en/blog/_posts/scheduler-plugin-reusing/index.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+There are some caveats, as usual. Arguments for the composite are needed, which basically means the args of the new plugin needs to contain and validate the args for the composite also. That may look rather redundant in the scheduler configuration, especially in case the args match exactly what the _NodeResourcesFit_ plugin is doing for individual node scoring, anyhow. The other caveat is, that the new plugin is then running the same node resource calculations another time, which isn't ideal.
+
+Putting these unfortunate drawbacks aside, getting the newly calculated node resource scores back to the new plugin is now only lacking a little bit of code:
+
+```go
+// NormalizeScore implements framework.ScoreExtensions.
+func (ce *CompositionExample) NormalizeScore(ctx context.Context, state *framework.CycleState, p *v1.Pod, scores framework.NodeScoreList) *framework.Status {
+    // logic for using and adjusting the incoming scores goes here
```
Comment (pohly) [inline content/en/blog/_posts/scheduler-plugin-reusing/index.md:57]: Indention is inconsistent here (tabs vs. spaces).

**Review by pohly — COMMENTED** [review]
Looks good to me after one (quick) pass through the text.

File: content/en/blog/_posts/scheduler-plugin-reusing/index.md (hunk 1 above)
Comment (uniemimu) [reply content/en/blog/_posts/scheduler-plugin-reusing/index.md]: fixed

**Comment by uniemimu** [comment]
Discussion in sig-scheduling resulted in a proposal to move this to https://www.kubernetes.dev/blog/ which to me makes sense.
@sftim or @onlydole could you help directing me with how to transfer this there?

**Comment by sftim** [comment]
> Discussion in sig-scheduling resulted in a proposal to move this to https://www.kubernetes.dev/blog/ 

1. Fork and clone https://github.com/kubernetes/contributor-site
2. Copy the text of the article into that site's blog directory. Make sure to adjust the content to match existing articles as the conventions are slightly different.
3. Finally, open a PR

**Comment by uniemimu** [comment]
Moved. Closing.

**Pull request closed without merging**